docs/installation.md

```diff
@@ -53,8 +53,9 @@ project's `.gitignore` or/and to your global `.gitignore`:
 echo ".metals/" >> .gitignore
 ```
 
-Note that you will need to invoke `metalsSetup` (or `semanticdbEnable`) whenever you close and
-re-open sbt. For a more persistent setup, keep reading.
+Note that in sbt 0.13 you will need to invoke `metalsSetup` (or `semanticdbEnable`) whenever you close and
+re-open sbt. For a more persistent setup, keep reading. In sbt 1 you don't need to do it because Metals will
+automatically invoke `semanticdbEnable` every time it connects to the sbt server.
```

Comment:
Is it possible to disable this behavior sbt 1? For example if `sbt.diagnostics=false` and `command=""`.

Reply:
I think this combination is a bit confusing. I'm hesitant to suggest adding another setting again after we removed `sbt.enabled`, but without it automatic connection to sbt server is currently unconditional.